integrations/operator/crdgen/testdata/protofiles/teleport/legacy/client/proto/authservice.proto

```diff
@@ -3104,31 +3104,6 @@ service AuthService {
   // DeleteAllSnowflakeSessions removes all Snowflake web sessions.
   rpc DeleteAllSnowflakeSessions(google.protobuf.Empty) returns (google.protobuf.Empty);
 
-  // CreateSAMLIdPSession creates web session with sub kind saml_idp used by the SAML IdP.
-  rpc CreateSAMLIdPSession(CreateSAMLIdPSessionRequest) returns (CreateSAMLIdPSessionResponse) {
```

Comment:
We should not delete the RPCs, it's "legal" in the buf breaking sense but it can be problematic if we end up readding them with different message definitions at a later time. If necessary we can clear out the messages by reserving all the existing fields (thankfully these are all enveloped).